apis/logging/v1/groupversion_info.go

```diff
@@ -35,7 +35,7 @@ var (
 	AddToScheme = SchemeBuilder.AddToScheme
 )
 
-// +kubebuilder:rbac:groups=console.openshift.io,resources=consolelinks;consoleexternalloglinks,verbs=get;create;update;delete
+// +kubebuilder:rbac:groups=console.openshift.io,resources=consolelinks;consoleexternalloglinks;consoleplugins,verbs=get;create;update;delete
```

Comment:
For me there's an error when using the ConsolePlugin as an OwnerRef:

```
cannot set blockOwnerDeletion if an ownerReference refers to a resource you can't set finalizers on
```

Adding this to the role has fixed this issue:

```suggestion
// +kubebuilder:rbac:groups=console.openshift.io,resources=consolelinks;consoleexternalloglinks;consoleplugins,verbs=get;create;update;delete
// +kubebuilder:rbac:groups=console.openshift.io,resources=consoleplugins/finalizers,verbs=update
```

Reply:
Fixed